packages/gutenframe/.gitignore

```diff
@@ -0,0 +1 @@
+/dist/
```

Comment:
Not necessary (although it will be if this moves to `/apps/`)

https://github.com/Automattic/wp-calypso/blob/cff012a9bab6f49946f61da95abe0b74710a2e93/.gitignore#L75-L76